packages/runtime/src/runtimes/react/controls/control.tsx

```diff
@@ -133,15 +133,15 @@ export function ControlValue({
         </ShapeControlValue>
       )
 
-    case ShapeV2Definition.type:
+    case GroupDefinition.type:
```

Comment:
We don't need to handle `ShapeV2Definition` rendering since there is no such control anymore.